tests/benchmarks/bench_assembly_levels.cpp

```diff
@@ -260,7 +260,7 @@ struct Kernel: public BakeOff
    GridFunction y;
 
    Kernel(AssemblyLevel assembly, int order, int N)
-   : BakeOff(assembly,order,N,VDIM,GLL), y(&fes)
+      : BakeOff(assembly,order,N,VDIM,GLL), y(&fes)
```

Comment:
Looks like we need to add `tests/benchmarks` to the directories where `make style` is applied.

Reply:
Actually there a few more sub-directories in `tests` that we should add: `convergence`, `mem_manager`, `par-mesh-format`. Also one sub-directory inside `tests/unit`: `enzyme`.